### PR TITLE
Jormungandr: Poi access point

### PR DESCRIPTION
**Pull request by azime**
Example [places/uri](https://github.com/hove-io/navitia-playground/pull/388)

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 1]
```diff
@@ -167,6 +167,45 @@ class PoiTypeSerializer(serpy.DictSerializer):
...
+class PoisSerializer(serpy.Field):
+    def as_getter(self, serializer_field_name, serializer_cls):
+        return lambda obj: self.make(obj)
+
+    def make(self, obj):
+        children = value_by_path(obj, 'properties.geocoding.children', [])
+        if not children:
+            return None
+
+        def make_children(child):
```
Comment (woshilapin) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: It creates only one child.
```suggestion
        def make_child(child):
```

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 2]
```diff
@@ -88,32 +88,32 @@ def generate_coord_id(self, obj):
...
+        'coord': {'lon': str(admin['coord']['lon']), 'lat': str(admin['coord']['lat'])},
+    }
+    zip_codes = admin.get('zip_codes', [])
+    if all(zip_code == "" for zip_code in zip_codes):
+        pass
+    elif len(zip_codes) == 1:
+        res['zip_code'] = zip_codes[0]
+    else:
+        res['zip_code'] = '{}-{}'.format(min(zip_codes), max(zip_codes))
+    return res
```
Comment (woshilapin) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: We can use https://github.com/hove-io/navitia/blob/0ed9608f23492eac15a9826ad01904ed46d13ccd/source/jormungandr/jormungandr/autocomplete/geocodejson.py#L72 instead.

**Review by pbench — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 3]
```diff
@@ -167,6 +167,45 @@ class PoiTypeSerializer(serpy.DictSerializer):
...
+                'coord': {'lon': str(child['coord']['lon']), 'lat': str(child['coord']['lat'])},
+                "type": "poi",
+            }
+            zip_codes = child.get('zip_codes', [])
+            if all(zip_code == "" for zip_code in zip_codes):
+                pass
+            elif len(zip_codes) == 1:
+                res['zip_code'] = zip_codes[0]
+            else:
+                res['zip_code'] = '{}-{}'.format(min(zip_codes), max(zip_codes))
```
Comment (woshilapin) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: We can use https://github.com/hove-io/navitia/blob/0ed9608f23492eac15a9826ad01904ed46d13ccd/source/jormungandr/jormungandr/autocomplete/geocodejson.py#L72 instead.

**Review by woshilapin — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py (hunk 1 above)
Comment (azime) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: Done

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py (hunk 2 above)
Comment (azime) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: Done

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py (hunk 3 above)
Comment (azime) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: Done

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py (hunk 2 above)
Comment (woshilapin) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: Well, not exactly, my comment is about the seven lines, not just one.

**Review by woshilapin — APPROVED** [review]

**Pull request merged**